backend/src/main/resources/db/changelog/db.changelog-master.yaml

```
@@ -2171,3 +2171,19 @@ databaseChangeLog:
       changes:
         sql: |
           GRANT SELECT ON TABLE ${database.defaultSchemaName}.patient_registration_link TO ${noPhiUsername};
+  - changeSet:
+      id: add-automated-corrections-api_user
+      author: adam@skylight.digital
+      comment: Adds the automated corrections api_user to the database if they do not already exist
+      preConditions:
+        - onSqlOutput: FAIL
+        - onFail: MARK_RAN
+        - sqlCheck:
+            sql: |
+              SELECT case when count(1) = 0 then 0 else 1 end FROM ${database.defaultSchemaName}.api_user
+              WHERE login_email='corrections-noreply@simplereport.gov';
```

```suggestion
              SELECT count(1) FROM ${database.defaultSchemaName}.api_user
              WHERE login_email='corrections-noreply@simplereport.gov';
```
Any reason not to just... get the count?